infra/app/app-config/env-config/environment-variables.tf

```
@@ -102,6 +102,10 @@ locals {
       manage_method     = "manual"
       secret_store_name = "/service/${var.app_name}-${var.environment}/pinwheel-api-token-sandbox"
     },
+    AZ_DES_PINWHEEL_ENVIRONMENT = {
+      manage_method     = "manual"
+      secret_store_name = "/service/${var.app_name}-${var.environment}/az-pinwheel-environment"
```

Let's keep the `secret_store_name` values matching the environment variable name.

```suggestion
      secret_store_name = "/service/${var.app_name}-${var.environment}/az-des-pinwheel-environment"
```